service/frontend/wrappers/metered/api_generated.go

```diff
@@ -331,10 +333,12 @@ func (h *apiHandler) PollForDecisionTask(ctx context.Context, pp1 *types.PollFor
 	tags := []tag.Tag{tag.WorkflowHandlerName("PollForDecisionTask")}
 	tags = append(tags, toPollForDecisionTaskRequestTags(pp1)...)
 	scope := common.NewPerTaskListScope(pp1.Domain, pp1.TaskList.GetName(), pp1.TaskList.GetKind(), h.metricsClient, metrics.FrontendPollForDecisionTaskScope).Tagged(metrics.GetContextTags(ctx)...)
-	scope.IncCounter(metrics.CadenceRequestsPerTaskList)
+	scope.IncCounter(metrics.CadenceRequestsPerTaskListWithoutRollup)
 	sw := scope.StartTimer(metrics.CadenceLatencyPerTaskList)
 	defer sw.Stop()
-	swPerDomain := h.metricsClient.Scope(metrics.FrontendPollForDecisionTaskScope).Tagged(append(metrics.GetContextTags(ctx), metrics.DomainTag(pp1.GetDomain()))...).StartTimer(metrics.CadenceLatency)
+	scopePerDomain := h.metricsClient.Scope(metrics.FrontendPollForDecisionTaskScope).Tagged(append(metrics.GetContextTags(ctx), metrics.DomainTag(pp1.GetDomain()))...)
+	scopePerDomain.IncCounter(metrics.CadenceRequests)
+	swPerDomain := scopePerDomain.StartTimer(metrics.CadenceLatency)
```

Comment:
can you put metric samples with their relevant tags in the PR description?

Reply:
I meant comprehensive list of metrics emitted in by this generated function and their tag granularity. Also mention which ones existed before and which ones are new

Reply:
updated